src/state/apis/nft/types.ts

```diff
@@ -3,9 +3,8 @@ import type { AssetId, ChainId } from '@shapeshiftoss/caip'
 import type { MediaUrl } from '../zapper/validators'
 
 export type NftCollectionType = {
-  // The AssetId of the collection, which doesn't exist as an assetSlice asset yet (https://github.com/shapeshift/web/pull/4555)
-  // Note, depending on the endpoint being hit, we may or may not be able to get an actual collection AssetId
-  id: AssetId | null
+  // The CAIP-22/29 of the collection
```

Comment:
for the collection, this would just be the CAIP-19

Reply:
wondering if we just call it `assetId` as well to avoid confusion

Reply:
But yeah good point about `assetId` - currently tunnel-visioned on this one so `id` is obvious to me as an AssetId (it couldn't be the item id since this is a collection), though that might be less obvious for someone else looking at this in a few weeks/months!

Reply:
> CAIP-22 is effectively a superset of CAIP-19 i.e the manually composed examples for cryptokitties (`eip155:1/erc721:0x06012c8cf97BEaD5deAe237070F9587f8E7A266d` / `eip155:1/erc721:0x06012c8cf97BEaD5deAe237070F9587f8E7A266d/771769`) are the same for both https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-22.md https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-19.md

I guess to me it just feels like it will never have the token `/tokenId` suffix on the collection so we can just avoid confusion since it will always resemble what we typically think of as a caip19 assetId. Or we can just remove the comment and not really try to think about it in those terms and just know it is some caip asset id.

Reply:
> But yeah good point about `assetId` - currently tunnel-visioned on this one so `id` is obvious to me as an AssetId (it couldn't be the item id since this is a collection), though that might be less obvious for someone else looking at this in a few weeks/months!

The other reason I think it would be good to use `assetId` is because these will be our primary key relations across assetSlice, and nftSlice (collection, nftItem). we can always have another `id` key for non caip format id if we find it useful (like the nft token `id`)

Reply:
Absolutely agree with this rationale - LSP renamed to `assetId` and removed the comment in b16801bab